src/libs/Rules.ts

```diff
@@ -20,7 +20,7 @@ export const validatePasswordLength = (
   params: IValidationParameter
 ): boolean => {
   const min = params.min || 0
-  return params.value.length > Number(min)
+  return params.value.length >= Number(min)
```

Comment:
This will hit the max length that we set. 

For example: 

The max input is 8 then the user input 8 characters it would hit the validation since it was conditioned to be greater than or equal to.

Reply:
I modified this because it doesn't match the validation error message. And I think we are talking about the minimum not maximum length of password here. 🤔 

Before this modification, if user correctly set a 8 chars password, it would give a wrong error message like 'password need to be at least 8 characters'. Since min was set to be 8, `params.value.length > Number(8)` without `=` operation would actually ask for at least 9 characters.

Reply:
I understand. I have confirm it.
Thank you very much.